library_management/src/main/java/com/group2/library_management/entity/Author.java

```diff
@@ -0,0 +1,25 @@
+package com.group2.library_management.entity;
+
+import java.time.LocalDate;
+import java.util.*;
+
+import jakarta.persistence.*;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+@Entity
+@Table(name = "authors")
+public class Author extends Auditable {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Integer id;
+    @Column(length = 50,nullable = false)
+    private String name;
+    private LocalDate dob;
+    private LocalDate dod;
+    private String biography;
+    @OneToMany(mappedBy = "author",cascade = CascadeType.ALL,orphanRemoval = true, fetch = FetchType.LAZY)
+    public List<AuthorBook> authorBooks = new ArrayList<>();
```

Comment:
```suggestion
    public List<AuthorBook> authorBooks;
```
thật ra ri là đủ nhé
ngoài ra,m các class Enyiyes em bổ sung `@Builder` nhé 
vào trong đỡ khai báo lại 

ngoài ra, dùng @Setter, getter hay là dùng 
```
@Data
@AllArgsConstructor
@NoArgsConstructor
```

NGoài ra, chị đang thấy sự ko thống nhất giữa các file, file AuthorBook lại dùng @Data, vậy class này và class bên dưới khác j nhau, sao lại có sự khai báo khác nhau 

Chị thấy khai báo khá ko thống nhất, em chekc lại nhé, ở dưới là 1 mẫu cơ bản phải có, em có thể thêm các khai báo khác nhé 


```
@Entity
@Table(name = "name_table_in_database")
@Data
@AllArgsConstructor
@NoArgsConstructor
@Builder
@FieldDefaults(level = AccessLevel.PRIVATE)
```